pkg/provider/provider.go

```diff
@@ -26,6 +26,7 @@ import (
 )
 
 const (
+	EKSRetryCount    = 100
```

Comment:
Maybe add a comment about EKS taking longer here.

Reply:
Only cluster creation takes time, but I don't know the exact reason for it.